src/native/corehost/hostpolicy/hostpolicy_context.cpp

```diff
@@ -55,11 +56,15 @@ namespace
     const void* STDMETHODCALLTYPE pinvoke_override(const char* libraryName, const char* entrypointName)
     {
 #if defined(_WIN32)
+        const char* hostPolicyLib = "hostpolicy.dll";
```

Comment:
Is the extension needed for hostpolicy, but not for `System.IO.Compression.Native` on Windows and `libhostpolicy` on Unix?

Reply:
Tracking issue: https://github.com/dotnet/runtime/issues/60080

Reply:
Good point, Thanks!